src/TableColumn/Tooltip.php

```diff
@@ -64,12 +65,12 @@ public function getDataCellHTML(Field $f = null, $extra_tags = [])
         ]);
     }
 
-    public function getHTMLTags($row, $field)
+    public function getHTMLTags(Model $row, $field)
     {
         // @TODO remove popup tooltip when null
-        $tooltip = $row->data[$this->tooltip_field];
+        $tooltip = $row->get($this->tooltip_field);
 
-        if ($tooltip === null || empty($tooltip)) {
+        if ($tooltip === null || $tooltip === '') {
```

Comment:
what if tooltip is `null`? actual BC change

Reply:
Looking, looking, still think `null` is checked

Reply:
oh, silly me :D :D :)